chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/TestingLabController.java

```diff
@@ -80,8 +78,8 @@
     }
 
     @ApiOperation(value = "Get details about a specific testing lab (ATL).",
-            notes = "The logged in user must have ROLE_ADMIN or have either read or"
-                    + "administrative authority on the testing lab with the ID specified.")
+            notes = "Security Restrictions: ROLE_ADMIN, ROLE_ONC or have either read or administrative authority "
+                    + "on the testing lab with the ID specified.")
```

Comment:
Is this really true? Can the public call `/atls` and get all the data for all ATLs, but can't call `/atl/1`?

Reply:
There is no security in the manager.  Documentation updated.